src/math/math_utils.cc

```diff
@@ -228,6 +227,7 @@ namespace rascal {
           }  // if (m_count == 0)
         }    // for (m_count in [0, l])
       }      // for (l in [0, lmax])
+      // harmonics *= std::sqrt(2.0);
```

Comment:
Why is this here? As a comment, it's extremely confusing - if it works without it, then just delete this line.

Reply:
OK, I think it's left over from testing, so definitely delete it.

Reply:
```suggestion
```